task_5_1.py

```diff
@@ -0,0 +1,19 @@
+"""Introduction to programming’: Task 2,
+   Kyshynska Taisiia"""
+
+print("""Introduction to programming’: Task 2
+         Kyshynska Taisiia""")
+
+import math
+
+eps = 1e-4  # точність
+n = 1  # номер члена ряду
+a = math.factorial(n) / (3 * n ** n)  # перший член ряду
+suma = a  # початкова сума
+
+while abs(a) >= eps:
+    n += 1
+    a = math.factorial(n) / (3 * n ** n)
+    suma += a
```

Comment:
take a look at the example:

from itertools import takewhile, count, repeat

row = (1 / pow(2, n) + 1 / pow(3, n) for n in count())
print(sum(list(takewhile(lambda x: x >= 1e-4, row))))